src/types/castAction.ts

```diff
@@ -1,10 +1,19 @@
 import type { TypedResponse } from './response.js'
 
-export type CastActionResponse = {
+export type CastActionFrameResponse = {
   /**
-   * HTTP response headers.
+   * Action path to the frame
+   *
+   * @example '/my-frame'
    */
-  headers?: Record<string, string> | undefined
+  action: string
```

Comment:
Is `action` the right term? It doesn't perform an "action" does it? I would rather `path` that specifies where the frame is located for the Cast Action.

Reply:
Good call, indeed it doesn't perform an action. Path is clearer.